scheduler/utilization_based_host_allocator.go

```diff
@@ -64,21 +64,33 @@ func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData HostAl
 		}
 
 		// add up total number of hosts needed for all groups
-		nNewHosts += n
+		nNewHostsRequired += n
+	}
+	allocationCalcDuration := time.Since(allocationCalcBeginsAt)
+
+	// Will at least distro.PlannerSettings.MinimumHosts be running once nNewHostsRequired are up and running?
+	minimumHostsThreshold := distro.PlannerSettings.MinimumHosts
+	nExistingAndRequiredHosts := nExistingHosts + nNewHostsRequired
+	nAdditionalHostsToMeetMinimum := 0
```

Comment:
Instead of defining this variable, can we compare the # of new hosts requested to the min, and increase that amount to meet the minimum if needed? It's currently a bit confusing to follow the arithmetic here

Reply:
As discussed 1:1 - leaving as is for logging reasons.